service-recipe/src/main/java/com/team13/servicerecipe/service/RecipeCommentService.java

```diff
@@ -58,9 +60,10 @@ public List<RecipeCommentDto> getCommentsByUserId(Long userId) {
                 .collect(Collectors.toList());
     }
 
-    //해당 레시피에 적힌 댓글 모두 불러오기
+    //해당 레시피에 적힌 댓글 모두 불러오기 대댓글 계층 구분
     public List<RecipeCommentDto> getCommentsByRecipeId(Long recipeId) {
         return recipeCommentRepository.findByRecipeId(recipeId).stream().
+                filter(comment -> comment.getParentComment() == null).
                 map(this::convertToDto).
                 collect(Collectors.toList());
```

Comment:
```suggestion
                .filter(comment -> comment.getParentComment() == null)
                .map(this::convertToDto)
                .collect(Collectors.toList());
```
크게 중요한 건 아니지만, 메서드를 이어서 작성할 때는 위와 같이 마침표를 앞에다 적어주시는 게 좋을 것 같습니다.
(보통 그렇게 사용하기도 하고, 해당 파일의 코드에서도 마침표를 앞에다 붙이는 컨벤션을 사용하고 있네욥)

Reply:
넵 알겠습니다! 수정하겠습니다!